Health/src/main/webapp/app/controller/dataframe/CancerData.js

```diff
@@ -20,31 +20,108 @@ function getColumnName(){
 	var DataFrame = dfjs.DataFrame;
 	  DataFrame.fromCSV('./DownloadableContent/CancerRate.csv').then(
 				 df => {
-				     //data = df.toArray();
-				     //console.log(typeof(data));
-				     var columns = df.listColumns();
-				     console.log('--------Column Names-------');
-				     console.log (columns);
-				     console.log('--------Column Distinct Values-------');
-				     for (var i = 0; i<columns.length; i++ ){
-				    	 console.log('--------------' + columns[i] + '------------');
-				    	 var distinctArray = df.distinct(columns[i]).toDict()[columns[i]];
-				    	 console.log (distinctArray);
-				     }
-				     df.distinct('column1')
-				     
-	                      /*for(var i=0; i<data.length; i++){
```

Comment:
if not required can you remove the commented code and push changes?
Lets merge it